perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -1167,11 +1173,15 @@ public List<Pair<String, String>> getAllBlockedLoginsInNamespaces(PerunSession s
 	}
 
 	@Override
-	public boolean isLoginBlocked(PerunSession sess, String login) {
+	public boolean isLoginBlocked(PerunSession sess, String login, boolean ignoreCase) {
 		Utils.notNull(login, "userLogin");
 
 		try {
-			return jdbc.queryForInt("select count(1) from blocked_logins where login=?", login) == 1;
+			if (ignoreCase) {
```

Comment:
i'd say let's be consistent about handling the globally blocked logins - apply the ignoreCase only to the namespaced logins and always match globally blocked logins case insensitively. would be nice to mention it in the javadocs too